performance/src/performance/timer.cpp

```diff
@@ -36,6 +41,7 @@ timer::~timer() {
     } else {
         pos->second += totalTime;
     }
+    nvtxRangePop();
```

Comment:
Move this before all of the code in the destructor. It may be either before the

```c++
const auto end = std::chrono::high_resolution_clock::now();
```

line, or just after it. (That could be debated actually.) But definitely not after everything else.

Reply:
Adjustment made